dm/config/task.go

```diff
@@ -314,8 +325,8 @@ func (c *TaskConfig) Decode(data string) error {
 
 // adjust adjusts configs
 func (c *TaskConfig) adjust() error {
-	if len(c.Name) == 0 {
-		return terror.ErrConfigNeedUniqueTaskName.Generate()
+	if !c.verifyTaskName() {
+		return terror.ErrConfigTaskNameNotValid.Generate()
```

Comment:
do we need add the task name in error log?

Reply:
I think the old error `ErrConfigTaskNameEmpty` or `ErrConfigNeedUniqueTaskName` is not proper, so I add new one.

Reply:
I mean add the task's name in error `ErrConfigTaskNameNotValid`'s log

Reply:
sure, [Done](https://github.com/pingcap/dm/pull/290/files#diff-b263153ff75f30e0c0f3fd51160160faR334)